pandas/core/reshape/merge.py

```diff
@@ -743,6 +776,47 @@ def get_result(self) -> DataFrame:
 
         return result.__finalize__(self, method="merge")
 
+    # TODO: perform conditional merge as optimized cython,
+    #       rather than chunked/filtered cross merge
+    def _get_chunked_conditional_merge_result(self):
+        from pandas.core.reshape.concat import concat
+
+        result_chunks = []
+        for chunk_left, chunk_right in self._chunk_pairs:
+            chunk_result = merge(
+                chunk_left,
+                chunk_right,
+                how="cross",
+                suffixes=self.suffixes,
+                indicator=self.indicator,
+            )
+
+            chunk_result_left = chunk_result.iloc[:, : len(chunk_left.columns)]
+            chunk_result_left.columns = chunk_left.columns
+            chunk_result_right = chunk_result.iloc[:, len(chunk_left.columns) :]
+            chunk_result_right.columns = chunk_right.columns
+
+            """
```

Comment:
Nit: Could you convert this into a block comment with `#` instead of `"""`

Reply:
done